sentry_sdk/client.py

```diff
@@ -507,13 +510,16 @@ def _prepare_event(
                 return None
 
             event = event_
-
             spans_delta = spans_before - len(event.get("spans", []))
             if is_transaction and spans_delta > 0 and self.transport is not None:
                 self.transport.record_lost_event(
                     "event_processor", data_category="span", quantity=spans_delta
                 )
 
+            dropped_spans = event.pop("dropped_spans", 0) + spans_delta # type: int
+            if dropped_spans > 0 or spans_delta > 0:
```

Comment:
Unless I'm missing something we don't need the `spans_delta` check here -- if `spans_delta` is greater than 0, then so is `dropped_spans`:

```suggestion
            if dropped_spans > 0:
```

Reply:
Thanks, that was a leftover.